spec/requests/portfolios_spec.rb

```diff
@@ -1,5 +1,10 @@
 describe 'Portfolios API' do
-  before { disable_tenancy }
+  around do |example|
+    disable_tenancy
+    example.call
+    Tenant.delete_all
```

Comment:
Why is this needed?  Are we creating records outside of a db transaction that get left around?